### PR TITLE
fix(category,navigation): breadcrumb URL not locatable

### PR DESCRIPTION
**Pull request by griest024**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: #1536 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: libs/navigation/driver/magento/src/transformers/navigation-transformer.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export class DaffMagentoNavigationTransformerService implements DaffNavigationTr
       id: breadcrumb.category_uid,
       name: breadcrumb.category_name,
       level: breadcrumb.category_level,
-      url: breadcrumb.category_url_key,
+      url: `/${breadcrumb.category_url_path}`,
```
Comment (damienwebdev) [inline libs/navigation/driver/magento/src/transformers/navigation-transformer.ts:43]: While maybe not in scope, this is missing suffix right?

Comment (griest024) [reply libs/navigation/driver/magento/src/transformers/navigation-transformer.ts:43]: Yeah and it's probably in scope as well, I'll add it

Comment (griest024) [reply libs/navigation/driver/magento/src/transformers/navigation-transformer.ts]: added and also added the category breadcrumbs

**Review by damienwebdev — APPROVED** [review]
lgtm

**Pull request merged**